src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
```

Comment:
You can keep Main locally but it shouldn't be committed unless explicitly asked for
you can use git rm --cached Main.java to rectify this